opm/output/data/Wells.hpp

```diff
@@ -58,6 +58,9 @@ namespace Opm {
                 reservoir_water   = (1 << 8),
                 reservoir_oil     = (1 << 9),
                 reservoir_gas     = (1 << 10),
+                productivity_index_water = (1 << 11),
+                productivity_index_oil   = (1 << 12),
+                productivity_index_gas   = (1 << 13),
```

Comment:
Hehe - we are soon running out of bits ;-)

Reply:
and more is coming...